Modules/_pickle.c

```diff
@@ -2276,7 +2291,10 @@ save_bytes(PicklerObject *self, PyObject *obj)
         else {
             /* Bypass the in-memory buffer to directly stream large data
                into the underlying file object. */
-            if (_Pickler_write_large_bytes(self, header, len, obj) < 0) {
+            if (_Pickler_write_large_bytes(self, header, len,
```

Comment:
What does this change? If `self->write == NULL`, It calls `_Pickler_Write(self, header, len) < 0)` followed by `_Pickler_Write(self, PyBytes_AS_STRING(obj), size)` exactly as above.

Reply:
But it first commit the current frame and temporary disable framing.